stats_api.py

```diff
@@ -195,8 +196,9 @@ def _get_speed(self, iface):
             iface.set_custom_speed(user_speed)
         return iface.speed
 
-    def _add_utilization(self, row, iface):
+    def _add_utilization(self, row):
```

Comment:
Please make sure that no other method calls `_add_utilization()` with the `iface` parameter.

Reply:
I checked and it's ok.